apps/site/assets/css/_page.scss

```diff
@@ -11,6 +11,27 @@
       border-bottom-width: 0;
     }
   }
+
+  .list-group-item {
+    border-top: unset;
+    border-top: solid $gray-lightest;
+    border-width: 1px 0 0 0;
+  }
+
+  .m-event__title {
+    a {
+      color: $black;
+      &:hover {
+        color: $brand-primary;
+      }
+    }
```

Comment:
There's a _tiny_ note on the mockup to update the original styling for these links to match this -- so you can move this `a {}` declaration under the `.m-event__title` in line 238 of `_events.scss`